create_stacksentinel_run_files.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python3
+########################
+# Author: Sara Mirzaee
+#######################
+
+import os
+import subprocess
+import sys
+import glob
+
+import argparse
```

Comment:
@2gotgrossman : 
https://www.google.com/search?client=safari&rls=en&q=argparse+optional+positional+arguments&ie=UTF-8&oe=UTF-8
```suggestion
import argparse
```